docs/design-docs/overview.md

```diff
@@ -0,0 +1,144 @@
+---
+id: overview
+---
+
+# Superbench Design
+
+## Goals
+
+Extend current tool to fit distribution test needs in cluster with 100 ~ 1000 nodes, make it modulable, easy to use and easy to scale up.
```

Comment:
SuperBench targets on providing a distribution test in cluster with 100 ~ 1000 nodes, and making it modulable, easy to use and easy to scale up. Therefore, SuperBench would like to provide a more simple and convenient way to: